wal/src/manager.rs

```diff
@@ -119,12 +119,28 @@ pub mod error {
     define_result!(Error);
 }
 
-use common_types::{MAX_SEQUENCE_NUMBER, MIN_SEQUENCE_NUMBER};
+use common_types::{table::TableId, MAX_SEQUENCE_NUMBER, MIN_SEQUENCE_NUMBER};
```

Comment:
These import statements seem weird to be put here, and how about put them in the beginning of the file?

Reply:
Done.